### PR TITLE
package yum: fix a name of test suite package for Percona Server 5.7

### PR DESCRIPTION
**Pull request by komainu8**
TODO: Test

### CODE REVIEW

**Comment by kou** [comment]
Can we do this in `case ${package} in ...`?

**Comment by komainu8** [comment]
Is https://github.com/mroonga/mroonga/pull/403/commits/7b24d9809eb3a0cf578be721074a1b6b71ed201d as expected?

**Comment by kou** [comment]
No. Here: https://github.com/mroonga/mroonga/blob/master/packages/yum/test.sh#L52-L68

File: packages/yum/test.sh [hunk 1]
```diff
@@ -43,7 +43,7 @@ REPO
     mroonga_package=$(echo "${package}" | \
                         sed 's/-server-\([0-9]*\)\.\([0-9]*\)/\1\2-community/g')
     service_name=mysqld
-    package_prefix=mysql-community
+    testpackage_name=mysql-community-test
```
Comment (kou) [inline packages/yum/test.sh:46]:
```suggestion
    test_package_name=mysql-community-test
```

**Pull request merged**